WordPressComKit/MeService.swift

```diff
@@ -0,0 +1,46 @@
+import Foundation
+import Alamofire
+
+public class MeService {
+    public init() {}
+    
+    public func fetchMe(completion: (Me?, NSError?) -> Void) {
+        Alamofire
+            .request(RequestRouter.Me())
+            .validate()
+            .responseJSON { response in
+                guard response.result.isSuccess else {
+                    completion(nil, response.result.error)
+                    return
+                }
+                
+                let json = response.result.value as? [String: AnyObject]
+                let me = Me()
+                me.ID = json!["ID"] as! Int
```

Comment:
I'd very much rather use `guard` than `!` for JSON parsing. In general, but specially in a library that others could use. You want to fail early if the API returns something unexpected, but you don't want to crash.